egs/tal_csasr/ASR/local/display_manifest_statistics.py

```diff
@@ -0,0 +1,96 @@
+# Copyright    2021  Xiaomi Corp.        (authors: Fangjun Kuang
+# 						                           Mingshuang Luo)
+#
+# See ../../../../LICENSE for clarification regarding multiple authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This file displays duration statistics of utterances in a manifest.
+You can use the displayed value to choose minimum/maximum duration
+to remove short and long utterances during the training.
+See the function `remove_short_and_long_utt()`
+in ../../../librispeech/ASR/transducer/train.py
+for usage.
+"""
+
+
+from lhotse import load_manifest
+
+
+def main():
+    paths = [
+        "./data/fbank/tal_csasr_cuts_train_set.jsonl.gz",
+        "./data/fbank/tal_csasr_cuts_dev_set.jsonl.gz",
+        "./data/fbank/tal_csasr_cuts_test_set.jsonl.gz",
+    ]
+
+    for path in paths:
+        print(f"Displaying the statistics for {path}")
+        cuts = load_manifest(path)
+        cuts.describe()
+
+
+if __name__ == "__main__":
+    main()
+
+"""
+Displaying the statistics for ./data/fbank/tal_csasr_cuts_train_set.jsonl.gz
+Cuts count: 1050000
+Total duration (hours): 1679.0
+Speech duration (hours): 1679.0 (100.0%)
+***
+Duration statistics (seconds):
+mean    5.8
+std     4.1
+min     0.3
+25%     2.8
+50%     4.4
+75%     7.3
+99%     18.0
+99.5%   18.8
+99.9%   20.8
+max     36.5
```

Comment:
There are the statistics about the durations among the train cuts.

Reply:
What about within-batch statistics? I am trying to understand if:
- a) the bucketing sampler is doing a poor job by putting cuts with very different durations together (so we can tune the settings of the sampler to do better), or 
- b) if the mini-batch cut durations are close to each other (in which case there is nothing we can do)